libraries/AC_AttitudeControl/AC_PosControl.h

```diff
@@ -253,97 +278,123 @@ class AC_PosControl
 
     /// set commanded position (cm), velocity (cm/s) and acceleration (cm/s/s) inputs when the path is created externally.
     void set_pos_vel_accel_NEU_cm(const Vector3p& pos_neu_cm, const Vector3f& vel_neu_cms, const Vector3f& accel_neu_cmss);
+    void set_pos_vel_accel_NEU_m(const Vector3p& pos_neu_m, const Vector3f& vel_neu_ms, const Vector3f& accel_neu_mss);
     void set_pos_vel_accel_NE_cm(const Vector2p& pos_ne_cm, const Vector2f& vel_ne_cms, const Vector2f& accel_ne_cmss);
+    void set_pos_vel_accel_NE_m(const Vector2p& pos_ne_m, const Vector2f& vel_ne_ms, const Vector2f& accel_ne_mss);
 
 
     /// Position
 
-    /// get_pos_estimate_NEU_cm - returns the current position estimate, frame NEU in cm relative to the EKF origin
-    const Vector3p& get_pos_estimate_NEU_cm() const { return _pos_estimate_neu_cm; }
+    /// get_pos_estimate_NEU_m - returns the current position estimate, frame NEU in cm relative to the EKF origin
+    const Vector3p get_pos_estimate_NEU_cm() const { return _pos_estimate_neu_m * 100.0; }
+    const Vector3p& get_pos_estimate_NEU_m() const { return _pos_estimate_neu_m; }
 
-    /// get_pos_target_NEU_cm - returns the position target, frame NEU in cm relative to the EKF origin
-    const Vector3p& get_pos_target_NEU_cm() const { return _pos_target_neu_cm; }
+    /// get_pos_target_NEU_m - returns the position target, frame NEU in cm relative to the EKF origin
+    const Vector3p get_pos_target_NEU_cm() const { return _pos_target_neu_m * 100.0; }
+    const Vector3p& get_pos_target_NEU_m() const { return _pos_target_neu_m; }
 
-    /// set_pos_desired_NE_cm - sets the position target, frame NEU in cm relative to the EKF origin
-    void set_pos_desired_NE_cm(const Vector2f& pos_desired_ne_cm) { _pos_desired_neu_cm.xy() = pos_desired_ne_cm.topostype(); }
+    /// set_pos_desired_NE_m - sets the position target, frame NEU in cm relative to the EKF origin
+    void set_pos_desired_NE_cm(const Vector2f& pos_desired_ne_cm) { _pos_desired_neu_m.xy() = pos_desired_ne_cm.topostype() * 0.01; }
```

Comment:
I guess it's a step too fix the interface so that we use p-type instead of floats?